spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherProperties.java

```diff
@@ -43,4 +71,16 @@ public void setModules(String[] modules) {
 		return modules;
 	}
 
+	@PostConstruct
+	public void deduceModuleArguments() throws Exception {
```

Comment:
The `ModuleLauncher` can iterate the array in any order it wants. For an external user, I would suppose it would be more intuitive to say
`MODULES=time,transformer,transformer` than `MODULES=transformer,transformer,time`.
It just reads better that way, that is all.